README.md

```diff
@@ -188,11 +188,12 @@ summary_num_results: 10
 
 #### Reranking (optional)
 
-Whether to use Vectara's [reranking](https://docs.vectara.com/docs/api-reference/search-apis/reranking) functionality. 
+Whether to use Vectara's [reranking](https://docs.vectara.com/docs/api-reference/search-apis/reranking) functionality. Note that re-ranking currently works for English language only, so if the documents in your corpus are in other languages, it's recommended to set this to "false"
```

Comment:
Typo: "re-ranking" -> "reranking"

Also, missing a period at the end of the sentence.